### ISSUE TITLE
Strange Windows color configurations after uninstall

### ISSUE THREAD

**Issue by ThunderWolfGithub** [Asteski/WinMac#19]
After uninstalling WinMac (version 0.6.0), my system color scheme was weird. Dragging on the desktop caused a brown colored selection to appear, and some menus had brown highlights as well. I narrowed this down to the registry HKCU\Control Panel\Colors, but changing it just has the values reverted after a restart. Do you know what is causing this? Your help is highly appreciated!

![image](https://github.com/user-attachments/assets/f6f60272-9ecb-41be-a368-d4042074dba0)
The desktop selection
![image](https://github.com/user-attachments/assets/c7f5ac6b-9ba7-4892-ba85-52af3a7bbc88)
File explorer selections are fine
![image](https://github.com/user-attachments/assets/9d4941ad-0508-4c04-b4b7-26fe1d2dae6d)
Some menu highlights are brown
![image](https://github.com/user-attachments/assets/527c47a9-935f-4490-9599-747136421c0f)
Text highlight is also brown

**Comment by Asteski** [comment]
@ThunderWolfGithub Hello, tell me exactly what you installed and uninstalled with WinMac.

**Comment by ThunderWolfGithub** [comment]
@Asteski The only other customization software I use is Rainmeter; I have been using that for 1.5+ years and never had this problem. When I installed WinMac, I only installed Nexus and StartAllBack. I had Start Menu set to classic and prompt style to winmac, as well as shell corner style to rounded and theme style to dark. 
When I uninstalled WinMac, I just ran uninstall.ps1 through pwsh with -EP Bypass as admin with no args.
I don't know if this is relevant, but I recently upgraded to Windows Insider Beta build 26100.1742, so I'll just state that out here for now.
Thanks for the prompt response! I might not reply because it's late and I'd prefer to get some sleep in lol. Will check back here first thing tomorrow!

**Comment by Asteski** [comment]
@ThunderWolfGithub Generally I don't support Insider builds because of their experimental and unpredicatable changes. I'll check this build and run some tests, but let me know if you'll encounter the same issue in stable build.

**Comment by ThunderWolfGithub** [comment]
So should I exit the program temporarily?

**Comment by Asteski** [comment]
> @Asteski So should I exit the insider program temporarily and test a stable build?

you can try, but it's your decision to make. I couldn't even recreate the issue on 4 machines, so I'm not sure I'll be able to help you here.

**Comment by jattttiiinnn** [comment]
I am getting this error while uninstalling winmac now I am really tired of this bcz I can't uninstall it 

PS C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh> .\uninstall.ps1 -EP Bypass
>>
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:599 char:36
+             $defender = Read-Host "Do you want to continue? (Y/n)"
+                                    ~~
Unexpected token 'Do' in expression or statement.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:599 char:38
+             $defender = Read-Host "Do you want to continue? (Y/n)"
+                                      ~
Missing statement body in do loop.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:640 char:4
+ For more information please visit WinMac GitHub page:
+    ~
Missing opening '(' after keyword 'for'.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:643 char:3
+ If you have any questions or suggestions, please contact me on GitHub ...
+   ~
Missing '(' after 'If' in if statement.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:647 char:15
+ ... ite-Host "----------------------------------------------------------- ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unexpected token '---------------------------------------------------------------------------"  -ForegroundColor Cyan
Write-Host
$restartConfirmation = Read-Host "Restart' in expression or statement.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:649 char:35
+ $restartConfirmation = Read-Host "Restart computer now? It's recommen ...
+                                   ~~~~~~~
Unexpected token 'Restart' in expression or statement.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:649 char:59
+ ...  computer now? It's recommended to fully apply all the changes (Y/n)"
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The string is missing the terminator: '.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:527 char:14
+         "12" {
+              ~
Missing closing '}' in statement block or type definition.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:391 char:26
+     switch ($app.Trim()) {
+                          ~
Missing closing '}' in statement block or type definition.
At C:\Users\jatin\Downloads\WinMac-main\WinMac-main\pwsh\uninstall.ps1:390 char:33
+ foreach ($app in $selectedApps) {
+                                 ~
Missing closing '}' in statement block or type definition.
Not all parse errors were reported.  Correct the reported errors and try again.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : UnexpectedToken

**Comment by Asteski** [comment]
@jattttiiinnn 
1. have you run script directly from pwsh filder or haev you used EasyLauncher.bat?
2. have run script from Windows Powershell or Powershell 7?

**Comment by jattttiiinnn** [comment]
yeah I directly used pwsh folder like cd folder direction and I use powershell

**Comment by akamienski** [comment]
@jattttiiinnn you should onle use EasyLauncher.bat and not scripts directly from pwsh folder.

**Comment by jattttiiinnn** [comment]
I can't understand easylauncher  is used for in
stalling how can i uninstall  can you explain me the whole procedure for uninstalling

**Comment by akamienski** [comment]
@jattttiiinnn https://github.com/Asteski/WinMac/wiki/Installation-Guide

**Comment by jattttiiinnn** [comment]
It's not clearly given steps in uninstalling process can you plss explain it ??

**Comment by akamienski** [comment]
@jattttiiinnn just run this command: _EasyLauncher.bat -uninstall_ or _EasyLauncher.bat -uninstallnogui_ from WinMac repo folder as an admin, and either choose full or specific WinMac components.

**Comment by jattttiiinnn** [comment]
thanks broo and It was a great experience using winmac
